.github/actions/wait-for-es-data/action.yml

```diff
@@ -0,0 +1,35 @@
+name: "Wait for ES Data"
+description: "Waits for data on ES indices"
+inputs:
+  index:
+    description: "Which ES index to look at"
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Wait for data
+      env:
+        ES_HOST: http://localhost:9200
```

Comment:
For the input index, it is better to set it as an environment variable and then use it in the inline script.
```
env:
  INDEX: ${{ inputs.index }}
```
...

```
URL="${ES_HOST}/${INDEX}/_search?size=0"
```
...
```
echo "Checking index: ${INDEX}"
```

Reply:
i don't see how this is better, it's just longer. Could you explain how making an input variable also an environment variable adds value?